kolibri/core/assets/src/views/UserTable/index.vue

```diff
@@ -79,35 +79,165 @@
+                    the normal text color in `style` (using `color`
+                    prop won't work for this purpose)
+                  -->
+                    <KLabeledIcon
+                      :icon="isCoach ? 'coach' : 'person'"
+                      :label="user.full_name"
+                      data-test="fullName"
+                      :style="{ color: $themeTokens.text }"
+                    />
+                    <UserTypeDisplay
+                      aria-hidden="true"
+                      :userType="user.kind"
+                      :omitLearner="true"
+                      class="role-badge"
+                      data-test="userRoleBadge"
+                      :class="$computedClass(userRoleBadgeStyle)"
+                    />
+                  </KRadioButton>
+                  <template v-else>
+                    <KLabeledIcon
+                      :icon="isCoach ? 'coach' : 'person'"
+                      :label="user.full_name"
+                      :style="{ color: $themeTokens.text }"
+                      data-test="fullName"
+                    />
+                    <UserTypeDisplay
+                      aria-hidden="true"
+                      :userType="user.kind"
+                      :omitLearner="true"
+                      class="role-badge"
+                      data-test="userRoleBadge"
+                      :class="$computedClass(userRoleBadgeStyle)"
+                    />
+                  </template>
+                </td>
+                <td
+                  class="visuallyhidden"
+                  data-test="userRoleLabel"
+                >
+                  {{ typeDisplayMap[user.kind] }}
+                </td>
+                <td
+                  data-test="username"
+                  :style="{ color: $themeTokens.text }"
+                >
+                  <span dir="auto">
+                    {{ user.username }}
+                  </span>
+                </td>
+                <template v-if="showDemographicInfo">
+                  <td class="id-col">
+                    <KOptionalText :text="user.id_number ? user.id_number : ''" />
+                  </td>
+                  <td>
+                    <GenderDisplayText :gender="user.gender" />
+                  </td>
+                  <td>
+                    <BirthYearDisplayText :birthYear="user.birth_year" />
+                  </td>
+                </template>
+                <td v-if="$scopedSlots.info">
+                  <slot
+                    name="info"
+                    :user="user"
+                  ></slot>
+                </td>
+                <td
+                  v-if="$scopedSlots.action"
+                  class="core-table-button-col"
+                >
+                  <slot
+                    name="action"
+                    :user="user"
+                  ></slot>
+                </td>
+              </tr>
+            </template>
+          </KradioButtonGroup>
```

Comment:
`Kradio...` -> `KRadio...` -

Reply:
fixed in [557c9d3](https://github.com/learningequality/kolibri/pull/12751/commits/557c9d3532363606879fecdfd12e62aef5358400)